source/lib/src/tabulate.cc

```diff
@@ -118,6 +120,12 @@ void deepmd::tabulate_fusion_se_a_cpu(FPTYPE* out,
         FPTYPE a5 = table[table_idx * last_layer_size * 6 + 6 * kk + 5];
         FPTYPE var =
             a0 + (a1 + (a2 + (a3 + (a4 + a5 * xx) * xx) * xx) * xx) * xx;
+        if (enable_se_atten) {
+          FPTYPE t = two_embed[ii * nnei * last_layer_size +
+                               jj * last_layer_size + kk];
+          var = var * t + t;
+        }
```

Comment:
Please check here - Codecov said it is not covered.

![image](https://github.com/deepmodeling/deepmd-kit/assets/9496702/24f209de-dd48-4076-a91c-3fdf96e14439)

Reply:
Actually it has been tested in source/lib/tests/test_tabulate_se_a.cc:701.
![image](https://github.com/deepmodeling/deepmd-kit/assets/47053538/1c4cc2b3-5ba2-4d63-8009-e2328039f280)

Reply:
I see. The issue might occur on the coverage side.